repositories.yaml

```diff
@@ -26,6 +26,7 @@ fenix:
     - org.mozilla.components:lib-crash
     - org.mozilla.components:support-sync-telemetry
     - org.mozilla.components:browser-engine-gecko-beta
+    - org.mozilla.appservices:logins
```

Comment:
This line also needs to be added to fenix-nightly below (I assume this data is collected by fenix nightly as well).

Reply:
Thanks, added.